File: backend/src/main/java/com/ll/hotel/domain/booking/payment/repository/PaymentRepository.java
package com.ll.hotel.domain.booking.payment.repository;

import com.ll.hotel.domain.booking.payment.entity.Payment;
import org.springframework.data.jpa.repository.JpaRepository;

public interface PaymentRepository extends JpaRepository<Payment, Long> {
    public boolean existsByMerchantUid(String merchantUid);
}

File: backend/src/main/java/com/ll/hotel/domain/booking/payment/repository/PaymentRepository.kt
package com.ll.hotel.domain.booking.payment.repository

import com.ll.hotel.domain.booking.payment.entity.Payment
import org.springframework.data.jpa.repository.JpaRepository

interface PaymentRepository : JpaRepository<Payment, Long> {
    fun existsByMerchantUid(merchantUid: String): Boolean
}